test/test_client.py

```diff
@@ -114,3 +114,29 @@ def test_make_offer(self):
         self.assertIsNotNone(response)
 
 
+    def test_make_offer_url(self):
+        sample_trade_url="https://steamcommunity.com/tradeoffer/new/?partner=314218906&token=sgA4FdNm"  # a sample trade url with escrow time of 15 days cause mobile auth not added
```

Comment:
too long line with comment, it should be max 120 chars with PEP-8, also what is a token parameter here.

Generally are you using pycharm to format code with PEP-8?